ui/app/components/app/connected-sites-list/index.scss

```diff
@@ -25,8 +24,10 @@
   }
 
   &__domain-name {
+    max-width: 215px;
     white-space: nowrap;
     overflow: hidden;
+    direction: rtl;
```

Comment:
I was wondering if this would be incorrect for right-to-left locales, but... I think origins are always supposed to be rendered left-to-right, which each individual chunk being right-to-left for RTL locales? 🤔 Something like that. i.e. it should always be `subdomain.registrable-domain.top-level-domain`, even if the chunks are RTL. I think. Assuming that's correct, then this seems fine too.

Reply:
Hm, I found this UX Stack Exchange post: https://ux.stackexchange.com/questions/55127/do-arabic-web-browsers-have-the-back-forward-buttons-reversed

This Bugzilla bug report: https://bugzilla.mozilla.org/show_bug.cgi?id=525831

And this IDNA RFC: https://tools.ietf.org/html/rfc5893

My takeaway from that is that we can't be sure? I'm disinclined to block over it, so I'll create an issue to investigate.

Reply:
I'm not sure we're talking about the same thing? None of those links seem to address the ordering of labels in the domain name, unless I missed something

Reply:
There's this image in the Bugzilla thread:
![image](https://user-images.githubusercontent.com/25517051/80392026-738d4780-8863-11ea-8239-afb1a3f622b5.png)